Build failure: jamesdsp

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

1. run `nix build github:nixos/nixpkgs/nixos-unstable#jamesdsp`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/291899616

### Relevant log output

```console
(ANSI formatting was removed from the output)

Running phase: qmakePrePhase
@nix { "action": "setPhase", "phase": "qmakePrePhase" }
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/7z6vyin837llzfswjg877av13fg7zgrz-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
QMAKEPATH=/nix/store/5axv6qs5lw2kbnq3b89ksplrc8ga71wc-qtbase-6.8.2-dev:/nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2:/nix/store/65105k00nj95gl5g004xmk4r4vyc555j-qtsvg-6.8.2-dev:/nix/store/d46mnr8pklv39>
qmake PREFIX=/nix/store/z2rx6q7ha7qgw0x3g2migsnd2pfarfhz-jamesdsp-2.7.0 NIX_OUTPUT_OUT=/nix/store/z2rx6q7ha7qgw0x3g2migsnd2pfarfhz-jamesdsp-2.7.0 NIX_OUTPUT_DEV=/nix/store/z2rx6q7ha7qgw0x3g2migsnd2pfarfhz-jamesds>
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
Info: creating stash file /build/source/.qmake.stash
qmake: enabled parallel building
qmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j16 SHELL=/nix/store/ki4if6b0w5bqv8dc5lrjp8xm7wjy9dlf-bash-5.2p37/bin/bash
cd libjamesdsp/ && ( test -e Makefile || /nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/bin/qmake -o Makefile /build/source/libjamesdsp/libjamesdsp.pro PREFIX=/nix/store/z2rx6q7ha7qgw0x3g2migsnd2pfarfhz>
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
make[1]: Entering directory '/build/source/libjamesdsp'
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
gcc -c -pipe -std=gnu11 -O2 -O2 -fPIC -w -D_REENTRANT -DLIBJAMESDSP_PLUGIN -DCUSTOM_CMD -Dprintf=redirected_printf -DANDROID_LOG_INFO=0 -DDEBUG -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Isubtree/Main/libjamesdsp/jni/james>
...skipping...
../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:504:17: note: '<anonymous>' declared here
  504 |         m_value = PromiseValue<T>{std::forward<V>(value)};
      |         ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In member function 'void QSharedPointer<T>::deref() [with T = AeqVersion]',
    inlined from 'QSharedPointer<T>::~QSharedPointer() [with T = AeqVersion]' at /nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/include/QtCore/qsharedpointer_impl.h:284:30,
    inlined from 'static QSharedPointer<T> QSharedPointer<T>::create(Args&& ...) [with Args = {const AeqVersion&}; T = AeqVersion]' at /nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/include/QtCore/qshar>
    inlined from 'QtPromisePrivate::PromiseValue<T>::PromiseValue(const T&) [with T = AeqVersion]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:95:35,
    inlined from 'void QtPromisePrivate::PromiseData<T>::resolve(V&&) [with V = const AeqVersion&; T = AeqVersion]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:504:17,
    inlined from 'void QtPromisePrivate::PromiseResolver<T>::resolve(V&&) [with V = const AeqVersion&; T = AeqVersion]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromiseresolver.h:62:34,
    inlined from 'void QtPromise::QPromiseResolve<T>::operator()(V&&) const [with V = const AeqVersion&; T = AeqVersion]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromiseresolver.h:110:27,
    inlined from 'AeqPackageManager::isUpdateAvailable()::<lambda(const QtPromise::QPromiseResolve<AeqVersion>&, const QtPromise::QPromiseReject<AeqVersion>&)>::<lambda()>::<lambda(AeqVersion)>::<lambda(AeqVersio>
    inlined from 'static void QtPromisePrivate::PromiseDispatch<void>::call(const Resolve&, const Reject&, Functor, Args&& ...) [with Resolve = QtPromise::QPromiseResolve<void>; Reject = QtPromise::QPromiseReject>
    inlined from 'QtPromisePrivate::PromiseHandler<AeqVersion, AeqPackageManager::isUpdateAvailable()::<lambda(const QtPromise::QPromiseResolve<AeqVersion>&, const QtPromise::QPromiseReject<AeqVersion>&)>::<lambd>
    inlined from 'constexpr _Res std::__invoke_impl(__invoke_other, _Fn&&, _Args&& ...) [with _Res = void; _Fn = QtPromisePrivate::PromiseHandler<AeqVersion, AeqPackageManager::isUpdateAvailable()::<lambda(const >
    inlined from 'constexpr std::enable_if_t<((bool)is_invocable_r_v<_Res, _Callable, _Args ...>), _Res> std::__invoke_r(_Callable&&, _Args&& ...) [with _Res = void; _Callable = QtPromisePrivate::PromiseHandler<A>
    inlined from 'static _Res std::_Function_handler<_Res(_ArgTypes ...), _Functor>::_M_invoke(const std::_Any_data&, _ArgTypes&& ...) [with _Res = void; _Functor = QtPromisePrivate::PromiseHandler<AeqVersion, Ae>
/nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/include/QtCore/qsharedpointer_impl.h:471:12: warning: '<unnamed>.QtPromisePrivate::PromiseValue<AeqVersion>::m_data.QSharedPointer<AeqVersion>::d' may be u>
  471 |     { deref(d); }
      |       ~~~~~^~~
../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h: In static member function 'static _Res std::_Function_handler<_Res(_ArgTypes ...), _Functor>::_M_invoke(const std::_Any_data&, _ArgTypes&& ...) [with _>
../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:504:17: note: '<anonymous>' declared here
  504 |         m_value = PromiseValue<T>{std::forward<V>(value)};
      |         ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In member function 'void QSharedPointer<T>::deref() [with T = QList<AeqMeasurement>]',
    inlined from 'QSharedPointer<T>::~QSharedPointer() [with T = QList<AeqMeasurement>]' at /nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/include/QtCore/qsharedpointer_impl.h:284:30,
    inlined from 'static QSharedPointer<T> QSharedPointer<T>::create(Args&& ...) [with Args = {QList<AeqMeasurement>}; T = QList<AeqMeasurement>]' at /nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/inclu>
    inlined from 'QtPromisePrivate::PromiseValue<T>::PromiseValue(T&&) [with T = QList<AeqMeasurement>]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:96:30,
    inlined from 'void QtPromisePrivate::PromiseData<T>::resolve(V&&) [with V = QList<AeqMeasurement>; T = QList<AeqMeasurement>]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:504:17,
    inlined from 'void QtPromisePrivate::PromiseResolver<T>::resolve(V&&) [with V = QList<AeqMeasurement>; T = QList<AeqMeasurement>]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromiseresolver.h:62:34,
    inlined from 'void QtPromise::QPromiseResolve<T>::operator()(V&&) const [with V = QList<AeqMeasurement>; T = QList<AeqMeasurement>]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromiseresolver.h:110:27,
    inlined from 'AeqPackageManager::getLocalIndex()::<lambda(const QtPromise::QPromiseResolve<QList<AeqMeasurement> >&, const QtPromise::QPromiseReject<QList<AeqMeasurement> >&)>' at subprojects/AutoEqIntegratio>
    inlined from 'QtPromise::QPromiseBase<T>::QPromiseBase(F) [with F = AeqPackageManager::getLocalIndex()::<lambda(const QtPromise::QPromiseResolve<QList<AeqMeasurement> >&, const QtPromise::QPromiseReject<QList>
/nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/include/QtCore/qsharedpointer_impl.h:471:12: warning: '<unnamed>.QtPromisePrivate::PromiseValue<QList<AeqMeasurement> >::m_data.QSharedPointer<QList<AeqMea>
  471 |     { deref(d); }
      |       ~~~~~^~~
../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h: In constructor 'QtPromise::QPromiseBase<T>::QPromiseBase(F) [with F = AeqPackageManager::getLocalIndex()::<lambda(const QtPromise::QPromiseResolve<QLis>
../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:504:17: note: '<anonymous>' declared here
  504 |         m_value = PromiseValue<T>{std::forward<V>(value)};
      |         ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In member function 'void QSharedPointer<T>::deref() [with T = AeqVersion]',
    inlined from 'QSharedPointer<T>::~QSharedPointer() [with T = AeqVersion]' at /nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/include/QtCore/qsharedpointer_impl.h:284:30,
    inlined from 'static QSharedPointer<T> QSharedPointer<T>::create(Args&& ...) [with Args = {AeqVersion}; T = AeqVersion]' at /nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/include/QtCore/qsharedpoint>
    inlined from 'QtPromisePrivate::PromiseValue<T>::PromiseValue(T&&) [with T = AeqVersion]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:96:30,
    inlined from 'void QtPromisePrivate::PromiseData<T>::resolve(V&&) [with V = AeqVersion; T = AeqVersion]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:504:17,
    inlined from 'void QtPromisePrivate::PromiseResolver<T>::resolve(V&&) [with V = AeqVersion; T = AeqVersion]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromiseresolver.h:62:34,
    inlined from 'void QtPromise::QPromiseResolve<T>::operator()(V&&) const [with V = AeqVersion; T = AeqVersion]' at ../3rdparty/qtpromise/include/../src/qtpromise/qpromiseresolver.h:110:27,
    inlined from 'AeqPackageManager::getLocalVersion()::<lambda(const QtPromise::QPromiseResolve<AeqVersion>&, const QtPromise::QPromiseReject<AeqVersion>&)>' at subprojects/AutoEqIntegration/AeqPackageManager.cp>
    inlined from 'QtPromise::QPromiseBase<T>::QPromiseBase(F) [with F = AeqPackageManager::getLocalVersion()::<lambda(const QtPromise::QPromiseResolve<AeqVersion>&, const QtPromise::QPromiseReject<AeqVersion>&)>;>
/nix/store/00dn0wh9afy98xja69p7i404jgznfkdk-qtbase-6.8.2/include/QtCore/qsharedpointer_impl.h:471:12: warning: '<unnamed>.QtPromisePrivate::PromiseValue<AeqVersion>::m_data.QSharedPointer<AeqVersion>::d' may be u>
  471 |     { deref(d); }
      |       ~~~~~^~~
../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h: In constructor 'QtPromise::QPromiseBase<T>::QPromiseBase(F) [with F = AeqPackageManager::getLocalVersion()::<lambda(const QtPromise::QPromiseResolve<Ae>
../3rdparty/qtpromise/include/../src/qtpromise/qpromise_p.h:504:17: note: '<anonymous>' declared here
  504 |         m_value = PromiseValue<T>{std::forward<V>(value)};
      |         ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: Leaving directory '/build/source/src'
make: *** [Makefile:73: sub-src-make_first] Error 2
```

### Additional context

This only started to manifest from the latest revision of `nixos-unstable`. The latest version number of the package on said channel is `2.7.0`.

If the build failure was not reproducible, my entire nix config is here: https://github.com/fdnt7/nix-config

Edit: I accidentally mentioned the wrong `rewine` account, my apologies.

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.13.6, NixOS, 25.05 (Warbler), 25.05.20250309.e3e32b6`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.12`
 - nixpkgs: `/nix/store/g4ppspdl4fy7hnp4jgjl4ll03v7i08w3-source`

### Notify maintainers

@pasqui23 @wineee 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/audio/jamesdsp/default.nix
+++ b/pkgs/applications/audio/jamesdsp/default.nix
@@ -1,6 +1,7 @@
 {
   copyDesktopItems,
   fetchFromGitHub,
+  fetchpatch,
   glibmm,
   gst_all_1,
   lib,
@@ -40,6 +41,14 @@ stdenv.mkDerivation (finalAttrs: {
     pkg-config
     copyDesktopItems
     wrapQtAppsHook
+  ];
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/Audio4Linux/JDSP4Linux/pull/241.patch";
+      hash = "sha256-RtVKlw2ca8An4FodeD0RN95z9yHDHBgAxsEwLAmW7co=";
+      name = "fix-build-with-new-pipewire.patch";
+    })
   ];
 
   buildInputs =

